### DISCUSSION TITLE
Add docs/package.lock.json to gitignore

### DISCUSSION THREAD

**Discussion by kotAPI** [rad-ui/ui#485] [Q&A] [answered]
Is it a norm? do we ship package.lock.json along with the project or let members have their own version of package.lock file

**Comment by kotAPI** [comment]
Some info from ChatGPT

<img width="826" alt="image" src="https://github.com/user-attachments/assets/44403a79-5411-4170-85cd-cd8377ab97b5">

**Comment by krau5** [comment, marked as answer]
> Is it a norm? do we ship package.lock.json along with the project or let members have their own version of package.lock file

Lock file (`yarn.lock`, `package-lock.json`, etc.) must be included in the project to ensure everyone uses the same dependency versions. If the lock file is removed:
1. Chances of unpredictable conflicts will be increased by far
2. Dependencies might differ across potential users/devs
3. If the dependencies are different, code's behavior is also unpredictable

**Reply by kotAPI** [reply]
Agreed, these were the reasons we hadn't deleted the lock file, this was suggested by one of the contributors, it had props - but the cons were much more - unpredictability is one of the biggest reasons
